helm_deploy/values-dev.yaml

```diff
@@ -7,14 +7,27 @@ generic-service:
   ingress:
     host: manage-adjudications-api-dev.hmpps.service.justice.gov.uk
 
+  serviceAccountName: hmpps-manage-adjudications-api-dev
+
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     OAUTH_ENDPOINT_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json
     PRISON_API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
 
   allowlist: null
 
+  namespace_secrets:
+    hmpps-domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAINEVENTS_ARN: "topic_arn"
```

Comment:
No longer need to provide `ACCESS_KEY_ID` or `SECRET_ACCESS_KEY` secrets as this is handled by IRSA